src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/files/main.js

```diff
@@ -237,18 +242,108 @@ define(function () {
         },
 
         /**
+         * render move media overlay
+         */
+        renderSelectCollection: function() {
+            this.sandbox.start([{
+                name: 'collections/components/collection-select@sulumedia',
+                options: {
+                    el: this.$find(constants.moveSelector)
+                }
+            }]);
+        },
+
+        /**
+         * starts overlay for move media
+         */
+        startMoveMediaOverlay: function() {
+            this.sandbox.emit('husky.datagrid.items.get-selected', function(ids) {
+                this.selectedIds = ids;
+                this.sandbox.emit('sulu.media.collection-select.open');
+            }.bind(this));
+        },
+
+        /**
+         * emit events to move selected media's
+         * @param collection
+         */
+        moveMedia: function(collection) {
+            var left = this.selectedIds.length;
+
+            this.sandbox.emit('sulu.media.collections.move-media', this.selectedIds, collection,
```

Comment:
Why not `sulu.media.media.move`? You are moving a media or file here, not a collection.